connectors/ModuleQ/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "host": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify host for your ModuleQ installation",
+          "displayName": "ModuleQ host",
+          "tooltip": "Host for ModuleQ installation"
+        }
+      },
+      "token": {
+        "oAuthSettings": {
+          "clientId": "fb414a67-b276-4e37-b460-31baf4a3ffad",
```

Comment:
Please remove your client id and replace with a text description.

Reply:
oops..fixed